1-mandatory/2-quotegenerator/style.css

```diff
@@ -1 +1,15 @@
 /** Write your CSS in here **/
+body{
+    background-color: blueviolet;
```

Comment:
This body background colour is a bit intense - maybe try finding something a bit more mellow